cl-forest.lisp

```diff
@@ -61,7 +65,8 @@ Return two values:
        ;;
        ;;    https://github.com/rigetticomputing/pyquil/blob/master/pyquil/forest.py#L234
        :accept "application/octet-stream"
-       :additional-headers `(("X-API-KEY" . ,*api-key*))
+       :additional-headers `(("X-API-KEY" . ,*api-key*)
+			     ("X-USER-ID" . ,*user-id*))
```

Comment:
here and elsewhere, make sure your editor doesn't use tabs.

in emacs, you can remove all tabs in the file by doing `C-x h M-x untabify RET`

Reply:
Done.